common/config/qryn.go

```diff
@@ -101,5 +149,22 @@ func parseURL(rawURL, apiKey, apiSecret string) (string, error) {
 		return parseURL(fmt.Sprintf("https://%s", rawURL), apiKey, apiSecret)
 	}
 
-	return fmt.Sprintf("https://%s:%s@%s", apiKey, apiSecret, u.Host), nil
+	return fmt.Sprintf("%s://%s:%s@%s", u.Scheme, apiKey, apiSecret, u.Host), nil
```

Comment:
if we just take the `u.Host`, we will silently drop any `u.Path` that the user might have set. I guess in most cases this is fine, but in clusters with internal routing this might break the export.

I am also fine with keeping it as is if you prefer and address this issue if someone bumps into it.

An alternative can be to modify the `u` object itself, and then use `u.String()` to make it a safe url string in idomatic way